projects/plugins/jetpack/modules/sso.php

```diff
@@ -563,6 +598,19 @@ public function jetpack_user_table_styles() {
 		<?php
 	}
 
+	/**
+	 * Style the Jetpack user rows and columns.
+	 */
+	public function jetpack_users_new_form_styles() {
+		?>
+		<style>
+			#createuser .form-field textarea {
+				width: 25em;
+			}
+		</style>
+		<?php
+	}
```

Comment:
This works! I wonder if we could leverage `jetpack-sso-admin-create-user.css`, introduced by `render_invitation_email_message` when in `user-new`? 

Maybe with a different wording we could use that function to add the Notice and enqueue all the CSS for that page.

Reply:
@escapemanuele - Thats a good idea. We can leverage that css file. I'll move my styling into `jetpack-sso-admin-create-user.css`.

I think it makes more sense to use `add_action( 'admin_print_styles-user-new.php', array( $this, 'jetpack_users_new_form_styles' ) );`, move my styling into `jetpack-sso-admin-create-user.css`, then enqeue the css in that function (function `jetpack_users_new_form_styles`).

This way we can separate the styling and the notice logic.

 WDYT?

Reply:
That makes totally sense for me @agrullon95 👍